src/deploy/campaign/environments.ts

```diff
@@ -253,6 +258,7 @@ const requires = (condition : boolean, message : string) => {
 };
 
 // No price spike before `minPrice` kicks in at `maxLength`
+// TODO: do we still need this?
 const validatePrice = (config : ICurvePriceConfig) => {
```

Comment:
no. but do we need other validations? we can't leave them to the contract at this stage.
the deploy code should validate all provided values so that contract does not revert in the middle of a campaign run.